server/src/lib/models/orders.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+import mongoose from 'mongoose';
+import Customers from './customers';
+const Schema = mongoose.Schema;
+
+const orderSchema = Schema({
+  orderDate: { type: Date, required: true, default: Date() },
+  orderTotal: { type: Number },
+  itemQuantity: { type: Number, required: true, default: 1 },
+  shippingOption: { type: String, default: 'ground', enum: ['ground', '2-day air', 'freight'] },
+  customer: { type: Schema.Types.ObjectId, ref: 'Customers', required: true },

```

Comment:
The ref string needs to match the name of the collection and it is case-sensitive. So, you need to either change this line to 'customers' OR change the name in your customers.js file. Make sure the same is done for orders.